src/lib/server/database/index.ts

```diff
@@ -490,6 +490,46 @@ export async function editTicketDueDate(
     }
 }
 
+export const enum AssignTicketPriorityResult {
+    /** Ticket priority successfully assigned. */
+    Success,
+    /** Ticket does not exist. */
+    TicketNotFound,
+    /** Provided priority id does not exist. */
+    InvalidPriority,
```

Comment:
For the sake of consistency with other `Result` enums...

> **Important**
> Please change all other references to the old names.

```suggestion
    /** Ticket priority successfully assigned. */
    Success,
    /** Ticket does not exist. */
    NoTicket,
    /** Provided priority id does not exist. */
    NoPriority,
```